tests/README.md

```diff
@@ -0,0 +1,63 @@
+# Tests
+
+These tests should ideally be run for every Pull Request, though due to the need to run `clasp login` before some commands, some tests are not run.
+
+
+# Configuration using Travis
+
+This is a bit more difficult, and why we include the `.clasprc.json.enc` file. The `.clasprc.json` file is used to authenticate many of the `clasp` commands, and is what is generated after using `clasp login`.
```

Comment:
Add an intro sentence as to what Travis is and how `clasp` uses it.

Reply:
👍